dbt_sugar/core/task/base.py

```diff
@@ -432,15 +436,23 @@ def find_model_schema_file(self, model_name: str) -> Tuple[Optional[Path], bool,
                         schema_file_exists = False
                         if schema_file_path.exists():
                             schema_file_exists = True
-                        return schema_file_path, schema_file_exists, is_already_documented
+                        return (
+                            schema_file_path,
+                            schema_file_exists,
+                            is_already_documented,
+                        )
 
                     if schema_file_path and model_file_found:
                         logger.debug(
                             f"'{model_name}' found in '{schema_file_path}' we'll update entry."
                         )
                         is_already_documented = True
                         schema_file_exists = True
-                        return schema_file_path, schema_file_exists, is_already_documented
+                        return (
```

Comment:
this project allows for a line length of 100 chars. I would rather leave the pre-commit hook take care of the formatting as it will standardise it for everyone. Could you please revert this change?